de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingProjectNegotiation.java

```diff
@@ -36,6 +37,8 @@
 
   private static final Logger LOG = Logger.getLogger(ArchiveIncomingProjectNegotiation.class);
 
+  private static final ActivityQueuer activityQueuer = new ActivityQueuer();
```

Comment:
Please add a comment that this is not the best place to instantiate this object here.